hardhat.config.ts

```diff
@@ -91,6 +93,26 @@ module.exports = {
       gas: 13000000,
       gasPrice: 20000000000,
     },
+    celo: {
```

Comment:
Could we just use the existing network instead of adding the same network with a different name?

Either rename `mainnet` or `celo`

Reply:
I renamed it to celo (I had to update also other parts of repo) since mainnet is taken by ETH mainnet